MailLibrary/Drivers/IDriver.php

```diff
@@ -134,10 +134,23 @@ function copyMail($mailId, $toMailbox);
 	 */
 	function moveMail($mailId, $toMailbox);
 
+
+	/**
+	 * Save mail content as eml to given file
+	 * @param int $mailId
+	 * @param $file
+	 * @throws DriverException
+	 */
+	function saveMail($mailId, $file);
+
+	function getHeaderInfo($mailId);
```

Comment:
What is this for? You can always use `$mail->getHeader(Mail::TO)` or `$mail[Mail::TO]` to get required headers. Is there anything I am missing?

Reply:
I was not able to get date of delivery from this. To I use: